source/indexes/create-search-index.txt

```diff
@@ -0,0 +1,89 @@
+.. _compass-create-search-index:
+
+============================
+Create an Atlas Search Index
+============================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. versionadded:: 1.40.0
+
+You can create Atlas Search indexes in |compass|. Atlas Search indexes
+let you query data in `Atlas Search
+<https://www.mongodb.com/docs/atlas/atlas-search/>`__. Atlas Search
+indexes enable performant text search queries by mapping search terms to
+the documents that contain those terms.
+
+About this Task
+---------------
+
+To create an Atlas Search index must meet the following requirements:
+
+- Hosted on `MongoDB Atlas <https://www.mongodb.com/docs/atlas/>`__
+
+- Atlas cluster tier of M10 or higher
+
+- MongoDB version of 7.0 or later
+
+Steps
+-----
+
+.. procedure::
+   :style: normal
+   
+   .. step:: Open the index creation dialog
+
+      From the :ref:`Indexes <indexes-tab>` tab, click the
+      :guilabel:`Create` button, then click :guilabel:`Search Index`.
+
+   .. step:: Specify a name for the index
+
+   .. step:: Specify the search index definition
+
+      |compass-short| provides templates for different kinds of search
+      indexes. To learn more, see
+      :ref:`search-index-definition-create-mongosh`.
+
+   .. step:: Click :guilabel:`Create Search Index`
+
+Example
+-------
+
+The following example definition creates a search index that excludes
+the ``directors`` and ``imdb.rating`` fields:
```

Comment:
This is slightly incorrect. It excludes storing the field contents in the index, but the field is still indexed. To give you a quick summary. In Atlas Search you can store the "processed field" to be indexed (for example, splits the words and lowercases them so you can find something independently of the case), and you can also store the "unprocessed field" (the source) so you can retrieve it from Atlas Search without looking up into MongoDB.

More info on storedSource here: https://www.mongodb.com/docs/atlas/atlas-search/stored-source-definition/

I would suggest removing the storedSource from the index definition to keep it simple and change the description to something along the lines:

The following example definition creates a search index that indexes all fields.

Reply:
Got it. Thanks for the clarification! Updating.